config/celery.py

```diff
@@ -0,0 +1,31 @@
+from __future__ import absolute_import
+import django
+from django.conf import settings
+import os
+import ssl
+from celery import Celery
+
+os.environ.setdefault('DJANGO_SETTINGS_MODULE', 'config.settings')
```

Comment:
Let's look into how to automate the production versus development environments for celery. Here is the Render documentation for Celery: https://render.com/docs/deploy-celery